src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenOverridingAndHiding.vb

```diff
@@ -553,7 +559,88 @@ BC30345: 'Public Shared Sub f([x As Integer = 0])' and 'Public Shared Sub f(ByRe
                ~
 </errors>)
         End Sub
+        <Fact()>
+        Public Sub OverloadingBasedOnOptionalParameters_B()
+            Dim useOpts = _ImplicitDefaultOptionalParameter_
+            If Not InternalSyntax.Parser.CheckFeatures(InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter, useOpts) Then
+                Assert.True(False, $"Feature{NameOf(InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter)} is not present")
+            End If
```

Comment:
@AlekseyTs I've refactor these checks to just the `assert`s, which is needed to validate that the feature is or isn't present.